pom.xml

```diff
@@ -27,6 +27,20 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api -->
+		<dependency>
```

Comment:
It's a good practice to avoid hardcoding the version of the dependency. Consider managing versions in a dependency management section or in properties.